provision-osx/tasks/install_homebrew.yml

```diff
@@ -0,0 +1,85 @@
+---
+
+- 
+  name: Check homebrew directory permissions
+  file:
+    path: "{{ homebrew_prefix }}"
+    owner: root
+    group: admin
+    state: directory
+    mode: 0775
+  become: yes
+
+- 
+  name: Check homebrew directory exists
+  file:
+    path: "{{ homebrew_install_path }}"
+    owner: "{{ ansible_user_id }}"
+    group: admin
+    state: directory
+    mode: 0775
+  become: yes
+
+- 
+  name: Install homebrew
+  git:
+    repo: "{{ homebrew_repo }}"
+    version: master
+    dest: "{{ homebrew_install_path }}"
+    update: no
+    accept_hostkey: yes
+    depth: 1
```

Comment:
This looks fine but we will need to document that github needs to be whitelisted in a proxied environment